portality/static/doaj/js/available_facetviews/admin.editorgroups.facetview.js

```diff
@@ -21,7 +21,7 @@ jQuery(document).ready(function($) {
             {'display':'Created Date','field':'created_date'},
             {'display':'Last Modified Date','field':'last_updated'},
             {'display':'Editor ID','field':'editor'},
```

Comment:
ah but there's more to it than that: I remember a problem where you look for account 01234567X and it doesn't turn up, despite the account existing (same string for the username). At the time it was to do with the fact the field (without .exact) was analysed, so ES was looking for a lowercase letter 'x' or something of the sort. We should decide if it should be an analysed field or not and go with one, but there are consequences to doing one or the other (e.g. with .exact it'd really need to match the query exactly).